src/api/resourceEmbedApi.ts

```diff
@@ -27,6 +27,7 @@ import { getEmbedsFromContent } from '../utils/getEmbedsFromContent';
 import { toArticleMetaData } from '../utils/toArticleMetaData';
 
 const accountId = getEnvironmentVariabel('BRIGHTCOVE_ACCOUNT_ID', '123456789');
+const playerId = getEnvironmentVariabel('BRIGHTCOVE_PLAYER_ID', '123456789');
```

Comment:
For at fallback skal ha nokke for seg.
```suggestion
const playerId = getEnvironmentVariabel('BRIGHTCOVE_PLAYER_ID', 'default');
```